lib/dai-plugin-mcd/src/ManagedCdp.js

```diff
@@ -0,0 +1,64 @@
+import { getIlkForCurrency } from './utils';
+import Maker from '@makerdao/dai';
+import ethAbi from 'web3-eth-abi';
+import assert from 'assert';
+const { getCurrency, DAI } = Maker;
+
+export default class ManagedCdp {
+  constructor(id, collateralType, cdpManager) {
+    this.id = id;
+    assert(collateralType, 'Must specify collateral type');
+    this.collateralType = collateralType;
+    this._cdpManager = cdpManager;
+  }
+
+  async getCollateralValue() {
+    return this.collateralType.wei((await this._getUrn()).ink);
+  }
+
+  async getDebtValue() {
```

Comment:
I think `art` times `ilk.rate` gives the debt value people are more likely to expect(?)

Reply:
Makes sense, will do—I haven’t been too careful about semantics here while setting up the skeleton, so good catch